ballerina/tests/generated_types.bal

```diff
@@ -0,0 +1,74 @@
+import ballerina/time;
+
+// TODO: Remove "public" from the generated types
+public enum Gender {
+    M,
+    F
+}
```

Comment:
Remove

Reply:
Fixed in [90235d8](https://github.com/ballerina-platform/module-ballerina-persist/pull/103/commits/90235d817df86c78d05ff4528baba5a18d2a584a)